src/ClusterBootstrap/params.py

```diff
@@ -58,6 +58,10 @@
         }
     },
 
+    "storage-manager": {
+        "port": 9092
```

Comment:
gpu-reporter is using 9092

Reply:
So it's better to use a different port even when they run on different machines?

Reply:
No. But I think they are running on the same machine? Both are on infra? No?